### PR TITLE
Update bescort.lic

### PR DESCRIPTION
**Pull request by aruthas**
I wanted to add functionality for the thieves guild in Shard.  A stringproc is needed to enter guild.  Also need to create a yaml setting for the Shard passsword

### CODE REVIEW

File: bescort.lic [hunk 1]
```diff
@@ -172,7 +172,7 @@ class Bescort
       ],
       [
         { name: 'shard_gates', regex: /shard/i, description: 'Use the Shard gates' }
-      ],
+      ], [{ name: 'thief_guild', regex: /thief_guild/i, description: 'Enter the Shard thieves guild' }],
```
Comment (rpherbig) [inline bescort.lic]: Please follow the formatting of the other parameters.

File: bescort.lic [hunk 2]
```diff
@@ -222,6 +222,8 @@ class Bescort
       take_rh_lang_barge
     elsif args.shard_gates
       use_shard_gates
+    elsif args.thief_guild
+	  enter_thief_guild
```
Comment (rpherbig) [inline bescort.lic]: Please follow the indentation of the other options (and it looks like there is a tab here, not spaces?)

**Comment by aruthas** [comment]
ok new at this, so I think I changed it to the same format.  At least it looks the same ;)

**Comment by rpherbig** [comment]
That looks great. Thank you for your contribution @aruthas !

**Pull request merged**